src/components/rating/rating.ts

```diff
@@ -343,13 +343,13 @@ export default class IgcRatingComponent extends FormAssociatedMixin(
       const { forward, backward } = this.clipSymbol(i, ltr);
       yield html`<igc-rating-symbol exportparts="symbol, full, empty">
         <igc-icon
-          collection="internal"
-          name="star"
+          collection="default"
+          name="star-filled"
```

Comment:
If the pattern is using underscores this should be `star_filled` and same for `star_outlined`, FYI